packages/analytics-core/test/plugins/destination.test.ts

```diff
@@ -236,7 +270,6 @@ describe('destination', () => {
       ];
       const send = jest.spyOn(destination, 'send').mockReturnValueOnce(Promise.resolve());
       const result = await destination.flush();
-      expect(destination.queue).toEqual([]);
```

Comment:
Why this and another one below are deleted?

Reply:
test `should filter dropped event and update the storage provider` is making sure the remove event from queue function.

Reply:
I see. Can we check `destination.queue` in tests with mocked http response like [this](https://github.com/amplitude/Amplitude-TypeScript/blob/802da9c85c35c6e1309812aff704efa3d5304933/packages/analytics-core/test/plugins/destination.test.ts#L644-L687) to make sure that the queue behaves as expected? I'm thinking if we could have added this check in https://github.com/amplitude/Amplitude-TypeScript/pull/673, we would not have the events accumulating issue.

Reply:
I added the destination.queue check in the test which has mocked HTTP response.